ramls/loan.json

```diff
@@ -134,9 +134,18 @@
           "properties": {
             "name": {
               "description": "Name of the item status",
-              "type": "string"
+              "type": "string",
+              "$ref": "schema/item-status-names.json"
+            },
+            "date": {
```

Comment:
Why is this property being introduced?

Reply:
It was just missed in schema definition, but it is being added in ItemSummaryRepresentation.

Reply:
Why was it included in the item representation? Was this deliberately done for all situations where that class was used, and therefore the bug is with the interface definition? Or should it not be included in all of those cases, and the bug is with the implementation?

Reply:
It is related to https://issues.folio.org/browse/CIRC-523. @VRohach, do we need `status.date` in the loan response?

Reply:
@bohdan-suprun In the current implementation, the item.status.date returning everywhere, where item uses. So, in that cases when item exists in the loan, item.status.date will be there (if it isn`t empty).

Reply:
Thanks for clarification, @VRohach. Will leave the declaration in json schema then.